src/cascadia/LocalTests_TerminalApp/TestHostApp/TestHostApp.vcxproj

```diff
@@ -29,6 +29,7 @@
   <Import Project="$(SolutionDir)\common.openconsole.props" Condition="'$(OpenConsoleDir)'==''" />
 
   <Import Project="$(OpenConsoleDir)\src\common.build.pre.props" />
+  <Import Project="$(OpenConsoleDir)\src\common.nugetversions.props" />

```

Comment:
We should garnet an understanding as to when the `targets` file isn't necessary but the `props` one is (and the other way around?)

Reply:
They should always be included as a pair.  If I missed the targets in one of these files that is a mistake.  (The various targets are opt-in by property setting so if a project doesn't need anything from them then there should be no side effects)

Reply:
I missed the second import in this file.  That was a mistake.  I did a scan of the rest and didn't find any other errors.  Good catch!